### PR TITLE
[JUJU-1623] Fix old series;

### PR DESCRIPTION
**Pull request by ycliuhw**
Fix old series;

### CODE REVIEW

File: jobs/ci-run/functional/amd64/bootstrap-constraints.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@
         default: 'amd64'
         description: 'Arch to build for.'
         name: BUILD_ARCH
+    - string:
+        default: jammy
+        description: ''
+        name: series
```
Comment (wallyworld) [inline jobs/ci-run/functional/amd64/bootstrap-constraints.yml:19]: job params are UPPER_CASE

File: jobs/ci-run/functional/proving-grounds-functional-tests.yml [hunk 2]
```diff
@@ -22,20 +22,22 @@
...
+            - name: nw-deploy-focal-amd64-lxd
+              current-parameters: true
+            - name: nw-deploy-jammy-amd64-lxd
               current-parameters: true
             - name: nw-deploy-bionic-eks
               current-parameters: true
             - name: nw-model-migration-amd64-vsphere
               current-parameters: true
-            - name: nw-deploy-bionic-vsphere
+            - name: nw-deploy-jammy-vsphere
```
Comment (wallyworld) [inline jobs/ci-run/functional/proving-grounds-functional-tests.yml:36]: no need for series in the job name, we don't care about that here
it's just a deploy to vsphere test

**Review by wallyworld — APPROVED** [review]
Looks like there was some existing code using lowercase `series` as a job param. We have been standardising on UPPERCASE for job params. Can we change it or is it threaded though such that it's not feasible.
Also, we don't want series in the job names unless we are explicitly testing something about that series. Most if not all current tests don't need it.

File: jobs/ci-run/functional/amd64/bootstrap-constraints.yml (hunk 1 above)
Comment (ycliuhw) [reply jobs/ci-run/functional/amd64/bootstrap-constraints.yml:19]: The lower case `series` parameter is passed from the root CI master to all the child jobs.
I can have a following PR the change it to UPPERCASE.

**Pull request merged**